### PR TITLE
chore(DatePicker): reduce invalid date warnings

### PR DESCRIPTION
**Pull request by joakbjerk**

### CODE REVIEW

**Review by langz — APPROVED** [review]
Thanks :) 

We had over 100 `convertStringToDate got invalid date:` here:
https://github.com/dnbexperience/eufemia/actions/runs/11793890236/job/32850375796

Now we have 55:
https://github.com/dnbexperience/eufemia/actions/runs/11794480235/job/32852165900?pr=4267

**Review by tujoworker — APPROVED** [review]
Nice 👏

File: packages/dnb-eufemia/src/components/date-picker/DatePickerCalc.ts [hunk 1]
```diff
@@ -251,7 +251,7 @@ export function convertStringToDate(
   date: string | Date,
   { date_format = null }: { date_format?: string | null } = {}
 ): Date {
-  if (date === null) {
+  if (date === null || date === undefined) {
```
Comment (tujoworker) [inline packages/dnb-eufemia/src/components/date-picker/DatePickerCalc.ts:254]:
```suggestion
  if (!date) {
```

Can we simply go for this?

Comment (joakbjerk) [reply packages/dnb-eufemia/src/components/date-picker/DatePickerCalc.ts:254]: Si

**Pull request merged**